src/main/blackbox/blackbox.c

```diff
@@ -788,9 +795,9 @@ static void writeIntraframe(void)
         blackboxWriteSignedVB(blackboxCurrent->motor[x] - blackboxCurrent->motor[0]);
     }
 
-    if (testBlackboxCondition(FLIGHT_LOG_FIELD_CONDITION_TRICOPTER)) {
-        //Assume the tail spends most of its time around the center
-        blackboxWriteSignedVB(blackboxCurrent->servo[5] - 1500);
+    for (int x = 0; x < MAX_SUPPORTED_SERVOS; x++) {
+        //Assume that servos spends most of its time around the center
+        blackboxWriteSignedVB(blackboxCurrent->servo[x] - 1500);
```

Comment:
You should use the set servo mid point instead of 1500: `servoParams(i)->middle`